examples/CMakeLists.txt

```diff
@@ -1,40 +1,55 @@
-add_executable(benchmarks_all benchmarks/main.cpp)
+
+file(GLOB_RECURSE tiny_dnn_headers "${CMAKE_SOURCE_DIR}/tiny_dnn/*.h")
```

Comment:
@edgarriba It is needed to add file items into example projects that can be seen in Solution Explorer  window of Visual Studio.